grails-app/controllers/org/pih/warehouse/api/ReplenishmentApiController.groovy

```diff
@@ -107,7 +108,7 @@ class ReplenishmentApiController {
         }
 
         if (!replenishment.replenishmentNumber) {
-            replenishment.replenishmentNumber = identifierService.generateOrderIdentifier()
+            replenishment.replenishmentNumber = identifierService.generateOrderIdentifier() + Constants.REPLENISHMENT_CODE
```

Comment:
Call it REPLENISHMENT_PREFIX and add it in the beginning (use the property from config)